### PR TITLE
autobump: bump more formulae

### PR DESCRIPTION
**Pull request by chenrui333**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: .github/autobump.txt [hunk 1]
```diff
@@ -993,10 +1027,10 @@ nixpacks
 nng
 node
 node_exporter
+node-build
 node-sass
 node@18
 node@20
-node
```
Comment (chenrui333) [inline .github/autobump.txt:999]: this is dup of L994

File: .github/autobump.txt [hunk 2]
```diff
@@ -8,6 +8,7 @@ activemq
 ada-url
 adb-enhanced
 adios2
+advancemame:
```
Comment (chenrui333) [inline .github/autobump.txt:11]: this seems wrong

Comment (p-linnane) [reply .github/autobump.txt]: Didn't we remove `advancemame` due to the licensing issue?

Comment (chenrui333) [reply .github/autobump.txt]: we did, shit. remove it now

**Review by p-linnane — APPROVED** [review]

**Pull request merged**